README.md

```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
 pip install bokeh
 ```
 
-For more information, refer to the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html).
+Check the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html) for further details.
 
 ## Resources
 
-Once Bokeh is installed, check out the [first steps guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
+Once Bokeh is installed, check out the [first step guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
```

Comment:
My major concern here is, that you can't include two plural words together, that sounds off,  I would like to suggest changing "first steps guides" to "first steps guide".

Reply:
@tcmetzger do you have any strong feelings here?

Reply:
Alternatively, a hyphen could help here "first-steps" to indicate that it is a collective noun

Reply:
A hyphen is sensible and it would reflect the intended meaning, but "First-steps" should be preceded by "guide" instead of "guides", do you think this makes sense?

Reply:
@VroomRk I have thought a little more about this and I think "first steps guides" is the best expression of the correct meaning.  For one there are *multiple* guides (nine of them in fact):
<img width="344" alt="Screen Shot 2022-08-11 at 11 24 58" src="https://user-images.githubusercontent.com/1078448/184211955-824b11dd-5db0-4755-87a7-09c344469847.png">

that all provide information about various and different first steps with Bokeh. I am not sure what prescriptivist rule  bars "plurals in a row", but:

* "first steps guides" is completely comprehensible and in fact does not even sound strange or unusual to my ear 
* English is malleable and forgiving, and sometimes the application of judgment is more important to convey the most accurate meaning

Also looking at the other page, I realize adding a hyphen only here would be inconsistent, so I think we should forego that as well. Please revert to the original "first steps guides" and then I think this will be ready to merge.

Reply:
Sure Bryan. I can understand that "first steps guides" don't change the intended meaning. I will revert the changes.